### ISSUE TITLE
more sectors options for adding PBDB

### ISSUE THREAD

**Issue by camiplata** [CatalogueOfLife/xcol#184]
Could it be possible to create different sectors for  [PBDB](https://www.checklistbank.org/dataset/1174/about)  base on extant and extinct taxa? AS it has one of the highest priorities is adding names from extant taxa (not necessarily wrong) for groups like birds or insects where there are more curated sources.

**Comment by mdoering** [comment]
It is sth we could add, yes. But maybe it doesn't have to be such a high priority?

**Comment by camiplata** [comment]
no, is not a priority. But then I would like to move PBDB towards the end of the priority lists
